styles/styles.css

```diff
@@ -255,3 +297,23 @@ main .section.highlight {
   margin: 0;
   padding: 40px 0;
 }
+
+main .section.home-section {
+  background-color: var(--section-color);
+  margin: 10px 0;
+  padding: 40px 0;
+}
+
+main .section.home-section p {
+  color: var(--text-color);
+}
+
+/* 404 error page */
+.image-block {
```

Comment:
Do not use any term called block when you create a additional class. This will lead to confusion with actual blocks.

Reply:
done